setup.py

```diff
@@ -29,7 +29,7 @@
     use_scm_version={
         "fallback_version": "unknown",
         "local_scheme": "node-and-date",
-        "write_to": "_version.py",
+        "write_to": "icepyx/core/_version.py",
```

Comment:
@weiji14 I needed this to be the write path for pip installations.  does this work on your end?